starter-core/src/main/java/io/micronaut/starter/feature/github/workflows/docker/templates/gradleGraalCEWorkflow.rocker.raw

```diff
@@ -0,0 +1,46 @@
+@import io.micronaut.starter.options.JdkVersion
+
+@args (
+JdkVersion jdkVersion
+)
+
+name: GraalVM CE CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@@v2
+      - uses: actions/cache@@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+            ~/.m2/repository
+          key: ${{ runner.os }}-gradle-test-${{ hashFiles('**/*.gradle') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-test-
+      - name: Setup GraalVM CE
+        uses: DeLaGuardo/setup-graalvm@@2.0
+        with:
+        @if (jdkVersion == JdkVersion.JDK_8) {
+            graalvm-version: 20.2.0.java8
```

Comment:
20.3 is coming next week, and we would need to update this versions here.

I think you can use `VersionInfo` from the feature and pass it as an argument to the template.